shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/ShuntingYardTableReplicationsMap.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.conf;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.hotels.shunting.yard.replicator.exec.conf.ct.ShuntingYardTableReplication;
+import com.hotels.shunting.yard.replicator.exec.conf.ct.ShuntingYardTableReplications;
+
+public class ShuntingYardTableReplicationsMap {
+  private final Map<String, ShuntingYardTableReplication> tableReplicationsMap = new HashMap<>();
+
+  public ShuntingYardTableReplicationsMap() {}
```

Comment:
Default constructor not needed, compiler will do this for you.

Reply:
I don't think, this happens in Java.

https://www.codebyamir.com/blog/java-essentials-the-default-constructor

Reply:
that links is when a super class is involved.

Reply:
If I remove this, I get compilation error in the class where I am using it.

The link above says: "If we do explicitly declare a constructor of any form, then this automatic insertion by the compiler won't occur. "

Not sure if I am doing something wrong here.

Reply:
Well, there's always a super class, even if just the implicit `Object`. I'm confused why this is needed, what's the compiler error?

Reply:
This is removed as it was being used on in the test. Both test and the default constructor are now removed.